spec/lib/video_info/providers/youtube_playlist_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
 
-describe VideoInfo::Providers::YoutubePlaylist do
+describe VideoInfo::Providers::YoutubePlaylist, :vcr do
   before(:each) do
-    api_key = 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0'
+    api_key = 'api_key_123'
```

Comment:
@joaocv3 hey, could you please share a bit more details about the reason for going to a fake API key here? (I'm curious).

Reply:
hey @thibaudgg! Sure

Mainly for security reasons, since a lot can be done with an enabled API key. Resources like videos and comments can be created or deleted with a valid API key